scheduler/postgresql/README.txt

```diff
@@ -0,0 +1,86 @@
+# For cook to function, it must have a postgresql server available. There are two approaches
+# 1. Run postgresql in docker (this script)
```

Comment:
I'm confused about why this is a `.txt` file if it's a script

Reply:
Its not a script. Its a text file with some bash commands available for copy&paste.

Reply:
In that case, thoughts on making this be a Markdown file with properly formatted code blocks?

Reply:
An excellent suggestion. Done.